docs/user/testing-metrics.md

```diff
@@ -38,12 +38,13 @@ This function will return a datatype appropriate to the specific type of the met
 assertEquals(5, GleanMetrics.Foo.UriCount.testGetValue())
 ```
 
-Note that each of these functions has it's visibility limited to the scope to only unit tests by making use of the `@VisibleForTesting` annotation, so the IDE should complain if you attempt to use them inside of client code.
+Note that each of these functions has its visibility limited to the scope of unit tests by making use of the `@VisibleForTesting` annotation, so the IDE should complain if you attempt to use them inside of client code.
 
 ## Testing metrics for custom pings
 
 In order to test metrics where the metric is included in more than one ping, the test functions take an optional `pingName` argument.
-This is the name of the ping that the metric is being sent in, such as `events` for the "events" ping, or `metrics` for the "metrics" ping.
+This is the name of the ping that the metric is being sent in, such as `events` for the ["events" ping](pings/events.md),
```

Comment:
In other document files, we're using "`events` ping" rather than ""events" ping". Should we be consistent with the other usages?

Reply:
Yeah, the phrasing here makes it a bit difficult. We keep repeating "events" and "metrics"

Reply:
I'm gonna use `"metrics"` to signal it's a string and change the other to make it consistent.